test/check-machines-create

```diff
@@ -1131,6 +1132,11 @@ class TestMachinesCreate(VirtualMachinesCase):
                     "cp {0}/cert.pem /usr/local/share/ca-certificates/cert.crt".format(self.test_obj.vm_tmpdir),
                     "update-ca-certificates"
                 ]
+            elif self.machine.image.startswith("arch"):
+                cmds += [
+                    "cp {0}/cert.pem /etc/ca-certificates/trust-source/anchors/cert.pem".format(self.test_obj.vm_tmpdir),
+                    "update-ca-trust"
+                ]
```

Comment:
This and the following arch specific code would fit better into the previous commit. This one is "test: skip failing tests due to missing dependencies", which really should just have the skipImage bits.